src/score_hv/harvesters/innov_netcdf.py

```diff
@@ -382,19 +382,20 @@ def get_data(self):
                                 region.name,
                                 region.grid,
                                 elevations[idx],
-                                ev_units,
+                                ev_unit,
                                 metric.name,
                                 stat,
                                 nc_vardata[idx]
                             )
 
                             harvested_data.append(item)
-
+            
+            '''
```

Comment:
why did you comment out the except? I think we want that, along with the try statement that Sergey noted

Reply:
There was broken code within the try block, which was difficult to debug due to (1) too much code within the try block, and (2) nonspecific except clause with obscure custom messaging. I found that simply removing the try/except block allowed for easier debugging and clearer code.